modules/_labs/breadcrumbs/react/lib/Breadcrumbs/Dropdown/Menu.tsx

```diff
@@ -0,0 +1,116 @@
+/** @jsx jsx */
+import React from 'react';
+import {css, jsx} from '@emotion/core';
+import {
+  borderRadius,
+  colors,
+  commonColors,
+  depth,
+  spacing,
+  type,
+} from '@workday/canvas-kit-react-core';
+
+// types
+import {DropdownContext} from './Provider';
+import {Breadcrumb} from '../types';
+// local components
+import {DropdownMenuItemLink} from './MenuItemLink';
+
+export interface DropdownMenuProps
+  extends React.HTMLAttributes<HTMLUListElement>,
+    Pick<DropdownContext, 'activeDropdownItem' | 'dropdownItems' | 'setActiveDropdownItem'> {
+  activeDropdownItemRef: React.RefObject<HTMLAnchorElement>;
+  resetFocus: () => void;
+  toggleActiveItemUp: () => void;
+  toggleActiveItemDown: () => void;
+}
+
+const menuWrapperStyles = css({
+  ...type.body,
+  ...depth[2],
+  backgroundColor: commonColors.background,
+  border: `1px solid ${colors.soap500}`,
+  borderRadius: borderRadius.m,
+  boxSizing: 'border-box',
+  display: 'inline-block',
+  marginTop: spacing.xxxs,
+  position: 'relative',
+  width: 'max-content',
+});
+
+const MenuWrapper = (props: React.HTMLAttributes<HTMLDivElement>) => {
+  return <div css={menuWrapperStyles} {...props} />;
+};
+
+const menuStyles = css({
+  boxSizing: 'border-box',
+  listStyle: 'none',
+  margin: `${spacing.xxs} 0`,
+  padding: 0,
+  width: 280,
+  wordBreak: 'break-word',
```

Comment:
Looks like there are some wrapping issues in IE11:
![image](https://user-images.githubusercontent.com/146020/94767547-7daaf900-0361-11eb-8b21-abe19b08436b.png)

According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break), `break-word` isn't supported by IE (and is in fact deprecated).

I think I'll need to take a look at the Select menu as well. I don't think I addressed this edge case of long items with no spaces.

Reply:
Good catch! Thank you. Looks like I can use [`overflowWrap: 'break-word'](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap) for IE.